build.gradle

```diff
@@ -123,7 +123,7 @@ check.dependsOn installDist
 
 dependencies {
     compile files("${System.properties['java.home']}/../lib/tools.jar")
-    compile 'org.broadinstitute:gatk:4.alpha.1-151-g76f78d2-20160624.165758-3'
+    compile 'org.broadinstitute:gatk:4.alpha.1-174-gbbd028b-20160701.172759-1'
```

Comment:
This snapshot appears to be missing the native PairHMM libraries -- please don't merge until this is fixed.